README.md

```diff
@@ -4,13 +4,17 @@
 ![Code Quality](https://github.com/bakdata/quick-cli/workflows/Code%20Quality/badge.svg)
 [![PyPI Version](https://img.shields.io/pypi/v/quick-cli?color=blue&label=PyPi%20Version)](https://pypi.org/project/quick-cli/)
 
+> [!WARNING]
+> Quick's CLI is deprecated and no longer actively maintained.
+> Visit our [user guide](https://bakdata.github.io/quick/0.8/user/) for more information.
```

Comment:
better version 'latest' then 0.8 or dev? see also other links.

Reply:
Since we are not releasing any new versions, the `0.8` would be `latest`

Reply:
This link should remain at 0.8 because we might remove the warning someday